src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -775,7 +775,8 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
 }
 
 static ucs_status_t
-uct_ib_mlx5_devx_query_lag(uct_ib_mlx5_md_t *md, uint8_t *state)
+uct_ib_mlx5_devx_query_lag(uct_ib_mlx5_md_t *md, uint8_t *state,
+                           uint8_t *port_select_mode)
```

Comment:
lets do output parameter type of `uct_dc_lag_port_select_t` and do type cast in scope of this function

Reply:
@evgeny-leksikov Please help below comment. ```port_select_mode``` doesn't align with ```uct_dc_lag_port_select_t```
https://github.com/openucx/ucx/pull/8382#discussion_r940316976

Reply:
> lets do output parameter type of `uct_dc_lag_port_select_t` and do type cast in scope of this function

not addressed

Reply:
@evgeny-leksikov Done, please help check it.